next-tavla/src/Admin/scenarios/Edit/index.tsx

```diff
@@ -23,6 +34,9 @@ function Edit({
     const [settings, dispatch] = useReducer(settingsReducer, initialSettings)
     const { addToast } = useToast()
 
+    const [editingTitle, setEditingTitle] = useState<boolean>(false)
+    const [title, setTitle] = useState<string>(initialSettings.title || 'Tavla')
```

Comment:
Trenger ikke en egen state for tittel. Du kan bruke settings objektet direkte, for oppdatering av staten kan du bruke reduceren (se `next-tavla/src/Admin/Scenarios/Edit/reducer.ts`).

Du kan lese mer om reducers og context her: https://react.dev/learn/scaling-up-with-reducer-and-context